frontend/src/features/public-form/components/FormFields/PublicFormSubmitButton.tsx

```diff
@@ -79,9 +86,15 @@ export const PublicFormSubmitButton = ({
     }
   }
 
+  const isSingleSubmissionOnlyModalOpen = hasSingleSubmissionValidationError
+  const onSingleSubmissionModalClose = () => {
+    setHasSingleSubmissionValidationError(false)
+  }
+
   return (
     <Stack px={{ base: '1rem', md: 0 }} pt="2.5rem" pb="4rem">
-      {isOpen ? (
+      // Prevent opening of multiple modals
```

Comment:
This will likely render the comment as text.

Reply:
strangely, it does not render it on my local dev even though i agree it should. 

i have refactored to `isPaymentsModalOpen && !isSingleSubmissionOnlyModalOpen` instead so the code is self-explanatory that there shouldnt be multiple modals